docs/other-api/adapter.md

```diff
@@ -117,7 +118,17 @@ exports.handler =
       };
 ```
 
-Here's an example with the simplified Cloudflare Workers API:
+Here's an example with the simplified Cloudflare Workers API (Module Worker / ES Modules syntax). This syntax supports additional Cloudflare Workers features, such as Durable Objects (you can access these bindings in the Remix context using the `getLoadContext(request, env, ctx)` option on `createEventHandler`). For more information on the difference between Module Workers and Service Workers, [see Cloudflare's docs][cloudflare-module-workers]):
+
+```ts
+import { createEventHandler } from "@remix-run/cloudflare-workers-esm";
+
+import * as build from "../build";
+
+export default { fetch: createEventHandler({ build }) };
+```
```

Comment:
I’m not sure if this is sufficient. Some users browsing the docs might want to know how to bind a Durable Object with an example. On the other hand, this is a good enough starting point and I think the community can fill it in.